js/views/homeVw.js

```diff
@@ -470,17 +470,21 @@ module.exports = baseVw.extend({
 
   searchItems: function(searchItemsText){
     if (searchItemsText){
+      var hashedItem = "#" + searchItemsText;
+      
+      window.obEventBus.trigger('saveVisitedTag', hashedItem);
```

Comment:
Could we please make the event type a more generic indicator of what is happening, rather than a directive to do something? Maybe something like 'searchingText'? Other pieces of code may want to hook into the same event.